package.json

```diff
@@ -29,5 +37,11 @@
     "@angular-devkit/schematics": "11.0.2",
     "electron": "^11.1.1"
   },
-  "schematics": "./dist/schematics/collection.json"
+  "schematics": "./dist/schematics/collection.json",
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
I would prefer not to use conventional commit messages by default. Since this is a new library, I would like to keep it simple for newcomers to contribute. We can still use this convention betwwen us though 😃